docker/standalone-ha.xml

```diff
@@ -101,9 +99,6 @@
             <logger category="org.jboss.as.config">
                 <level name="DEBUG"/>
             </logger>
-            <logger category="org.jgroups">
```

Comment:
This wasn't found in the 'new' `docker/standalone-ha.xml` when i did the build. Should I put it back?